CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 🚨 *Breaking Changes*
 
+* Removed model classes that used to define the shape of old `orq` CLI command outputs.
```

Comment:
Suggestion: to me, `old` doesn't fully communicate `no longer used`, how would you feel about: 

```suggestion
* Removed model classes that used to define the shape of defunct `orq` CLI command outputs.
```